terraform/root_common.tf

```diff
@@ -0,0 +1,307 @@
+module "global_parameters" {
```

Comment:
Is it worth splitting this out a little bit, as it a pretty big file, 307 lines? Wondering if it make sense to put the child node configuration in separate files: .../terraform/jenkins_nodes/jenkins_npm_node.tf

Not, sure might just make it too complicated, though not comfortable with such a large terraform file

Reply:
If I put them in a subdirectory then it counts as another module and I'll have to import them in. You're right about splitting the file though so I've moved them into their own files in the terraform directory. Is that better do you reckon?

Reply:
Have you done this?

Reply:
Yeah, there's a load of new files called `jenkins_*_node.tf` and the `root_common.tf` file is down to 128 lines.